src/Task/Test/Response.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace TYPO3\Surf\Task\Test;
```

Comment:
Move model to model namespace
```suggestion
namespace TYPO3\Surf\Domain\Model;
```